packages/manager/modules/pci/src/projects/project/storages/cold-archive/containers/delete-container/translations/Messages_fr_FR.json

```diff
@@ -4,9 +4,7 @@
   "pci_projects_project_storages_containers_container_cold_archive_delete_terminate_input_label": "Entrez \"TERMINATE\" dans le champ ci-dessous pour confirmer",
   "pci_projects_project_storages_containers_container_cold_archive_delete_submit_label": "Confirmer",
   "pci_projects_project_storages_containers_container_cold_archive_delete_cancel_label": "Annuler",
-  "pci_projects_project_storages_containers_container_cold_archive_delete_archive_erase_message": "Toutes les données sur le conteneur d'archive {{containerName}} seront perdues.",
-  "pci_projects_project_storages_containers_container_cold_archive_delete_archive_status_archived_restored_warning": "Votre archive {{containerName}} contient encore {{objetsCount}} objet(s).</br>Vous êtes sur le point de vider l'archive de tout son contenu.",
-  "pci_projects_project_storages_containers_container_cold_archive_delete_archive_status_none_description": "Votre archive ne pourrait pas être supprimer via le manager. Merci d'utiliser le AWS CLI pour cet effet.",
+  "pci_projects_project_storages_containers_container_cold_archive_delete_archive_status_archived_or_restored_warning": "Votre archive {{containerName}} contient encore {{objetsCount}} objet(s).</br>Pour supprimer votre archive, il faut d'abord supprimer les objets. Consultez notre <a class=\"oui-link oui-link_icon\" href=\"{{ docUrl }}\" target=\"_blank\" rel=\"noopener\">guide pour gérer les objects d'une archive avec AWS CLI</a>.",
```

Comment:
**nitpick:** Split the sentence into 2 and avoid using `<br>` in translation files.

Reply:
Done.